app/Resources/views/macro/messages.html.twig

```diff
@@ -0,0 +1,25 @@
+{#
+   Macros are comparable with functions in regular programming languages.
+   They are useful to put often used HTML idioms into reusable elements
+   to not repeat yourself.
+   See http://twig.sensiolabs.org/doc/tags/macro.html
+#}
+{% macro render_flash_messages() %}
+    {% from _self import alert %}
+
+    <div class="messages">
+        {% for flash_message in app.session.flashBag.get('success') %}
```

Comment:
this will start a session even if the user has no session yet. You should wrap this in a ``if app.session.started`` check